packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -255,14 +256,21 @@ const AnalyticalTable: FC<TableProps> = forwardRef((props: TableProps, ref: Ref<
     }
   }, []);
 
+  const updateTableClientWidthDebounced = useCallback(
+    debounce(() => {
+      updateTableClientWidth();
+    }, 500),
+    []
+  );
+
   useEffect(() => {
     // @ts-ignore
-    const tableWidthObserver = new ResizeObserver(updateTableClientWidth);
+    const tableWidthObserver = new ResizeObserver(updateTableClientWidthDebounced);
```

Comment:
```suggestion
    const tableWidthObserver = new ResizeObserver(debounce(() => {
      updateTableClientWidth();
    }, 500));
```
I would prefer this inline solution here because otherwise we will have two duplicated methods on top level which are basically doing similar things.